### PR TITLE
[12.0] [l10n_br_fiscal] lint

### PR DESCRIPTION
**Pull request by rvalyi**
more l10n_br_fiscal lint

@mileo as you can see, most of the code we still had to lint heavily comes from your company KMEE and was approved and pushed by you here https://github.com/OCA/l10n-brazil/pull/928
along with lot's of other very shitty untested code (models/mdfe, models/closing.py). You ended up fixing some of it, but 1 year later attachment.py was still all shitty...

This is really a problem you abuse your seat in the project PSC to push that kind of unsollicited code in the OCA and specially if you push it inside a critical module we wrote at 90% over many years without even getting our approval!! Please read again your code of l10n_br_fiscal/models/attachments.py before I tried to do something about it (hard to do anything as it is unused and untested code...)
https://github.com/akretion/l10n-brazil/blob/dc2436358509f77c3a8e51affc7bdf98884eb936/l10n_br_fiscal/models/attachment.py

Come one, is this 2020/2021 OCA quality code?? In which other OCA repo are such "contributions" accepted?
you have tests about variables like `value` or `config_ids` or `active_id` or `active_model` that you affect 3 lines before so your tests are useless (except from bloating the complexity beyond the flake8 OCA limits) and then you do nothing with these variables...

Moreover this kind of untested feature could perfectly be added in other modules. This way people could choose if they like your implementation or... may be not! But forcing us to bloat it inside the l10n_br_fiscal module and maintain it is not nice! Sorry to tell it loud and in plain English, but sadly you have been warned several times and you play it like you don't get it...

From what I saw, attachment,py seems somewhat tested in your other PR https://github.com/OCA/l10n-brazil/pull/1355/files so may be you can test if my refactor attempt didn't break it.
I made e comment BTW because I doubt your code works https://github.com/OCA/l10n-brazil/pull/1355/files#r645291763

### CODE REVIEW

**Comment by rvalyi** [comment]
cc @gabrielcardoso21 @luismalta @marcelsavegnago please you guys look at this code https://github.com/akretion/l10n-brazil/blob/dc2436358509f77c3a8e51affc7bdf98884eb936/l10n_br_fiscal/models/attachment.py#L88 and tell me if I'm wrong to be pissed of about this unsollicited code from KMEE in our module. At the time #928 was pushed without our approval, the code of the whole PR had a similar quality and lack of test/usage (hey we just saved KMEE from their own dead fork and we are not even asked for pushing the same shit in the OCA https://github.com/kmee/l10n-brazil/tree/10.0-develop ?!? )

**Comment by rvalyi** [comment]
Also, to make it straight: many times @mileo I tell you I disagree with datamodels (like the NFE field duplication you tried to push) or design decisions like the lack of modularity and god modules like you did in your v10/v11 dead KMEE fork. All right this is more subjective so eventually project's newcomers cannot see who is right or why we feel pissed of with your initiatives.

But such code as from the PR #928 and is still largely broken 1 year after you included it, is not subjective, it's shitty code by all objective metrics. So just because you keep behaving like you are the great master of the project that I dare point people to the things your company pushes in the project and you approve and you push on your own. And please be welcome to victimize yourself because there is more to come...

**Review by renatonlima — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Comment by renatonlima** [comment]
Teve um erro somente no terceiro job relacionado ao teste do test_website_sale_tour https://travis-ci.com/github/OCA/l10n-brazil/jobs/511504593#L4639

@mileo Seria bom investigar esse erro pois constantemente ele aparece...
